### DISCUSSION TITLE
Wisdom of Years (Talent) El-Aurian

### DISCUSSION THREAD

**Discussion by Captain-Harker** [bcholmes/StarTrek2d20#181] [Ideas]
Wisdom of Years: Your centuries of lifetime experience grant you an extra Talent, an extra Focus, as well as 1 added to your scores of Attribute and Discipline (i.e. total of 17 instead of 16 for Attributes, and total of 57 instead of 56 for Disciplines.)  This reflects many centuries, or thousands of years, of living experience.  (TNG, SNW, El-Aurians)

**Comment by Felderburg** [comment]
That seems a bit overpowered... also, I'm not entirely sure if @bcholmes is including random unofficial talents/species, or just stuff from Continuing Mission. Either way, I think submitting Talent ideas should have its own thread, not a thread for every talent; I also think it's something you might consider doing on the Modiphius forums or discord, or submitting to Continuing Mission, instead.

**Reply by Captain-Harker-613** [reply]
I actually found the "Wisdom of Years" online somewhere...somebody else has an El Aurian character!  I didn't consider adding +1 to Attributes and Disciplines to be "overpowered."  Guinan may be very wise, and have hidden talents, but she also nearly died in Time's Arrow.  My NPC (used to be PC) Dr. Jaryd Harker, is Guinan's godson.  He was only 12 when Aurelia was destroyed by the Borg, and the surviving El Aurians ended up on Earth.  He chose the name "Harker" for an El Aurian Listener, and grew up to become a medical doctor.  200 years later, he joined Starfleet.  A decade after that was the mission where Twilight had her bridge blown away, and Harker was the last senior officer left aboard, a Commander.  "Required to Take Command."  Upon return to Earth, Admiral Uhura and Admiral McCoy promoted him to captain, and permanently kicked him upstairs to the Center Seat!  That's when he/I insisted of the Gamemaster on a refit of the ship, since she needed a new bridge module, anyway.  Most of Deck 2 became an extensive Sickbay complex, thus, the Advanced Sickbay talent.

**Comment by bcholmes** [comment]
I think I'd prefer it if unofficial talents/species had a source such as Continuing Missions. Continuing Missions have [two ](https://continuingmissionsta.com/2020/12/30/player-species-el-aurian/)[write-ups](https://continuingmissionsta.com/2017/09/15/ten-forward-friday-el-aurian/) for El-Aurians, and the above talent is similar to (but not as powerful as) a "Wisdom of Years" talent for El-Aurians on CM. There's also a Temporal Awareness talent.

**Comment by Captain-Harker-613** [comment]
It seems to me that "Advanced Sickbay" should automatically include the Emergency Medical Hologram Talent, that way one of the valuable 3 Starship Talent slots aren't taken up by the EMH talent.  What do you folks think?  (I only just got the Second Edition, and haven't yet had a chance to thoroughly read through it, as yet.)  I know for a fact that the Nova Class ship has a Waverider shuttle, equipped with a very small sickbay, but that takes up one of the 3 Starship Talent slots!  As it stands right now, USS Twilight / aka USS Canary Islands, has 3 Talents: Advanced Sensor Suites, Emergency Medical Hologram, and Advanced Sickbays.  I put "Waverider Shuttle/Captain's Yacht" under "Special Rules."  (As designed by my friend Falke, the Waverider is equipped with the same sickbay in her aft, identical to the one aboard the Delta Flyer.)  
![Twilight STATS (refit) STM2](https://github.com/user-attachments/assets/35dd3251-5e17-4359-8b5f-14b3acfd2336)
